jota/src/main/java/jota/IotaAPI.java

```diff
@@ -602,7 +603,7 @@ public GetBalancesAndFormatResponse getBalanceAndFormat(final List<String> addre
             long balance = Long.parseLong(balances.get(++i));
 
             if (balance > 0) {
-                final Input newEntry = new Input(address, balance, start + i, security);
+                final Input newEntry = new Input(seed, address, balance, start + i, security);
```

Comment:
This call doesn't validate that an address belongs to the seed and a seed shouldn't be necessary here.

Two options exist from my point of view:
- Wrap seed in a `java.util.Optional` 
- Create a `InputWithSeed` class that extends Input and adds the seed property.